usercommandmgmt/usercommandmgmt.py

```diff
@@ -26,8 +28,10 @@ def __init__(self, bot):
             with open(saveFile, "w+") as f:
                 empty = {"db": []}
                 json.dump(empty, f)
-
         self.activeDb = Database(saveFile)
+        intents = discord.Intents.default()
+        intents.reactions = True
+        self.create_comamnd_queue = []
```

Comment:
nit: Here and everywhere else this is used, spelling
```suggestion
        self.create_command_queue = []
```